.github/workflows/main.yml

```
@@ -39,4 +39,5 @@ jobs:
         run: echo "::notice::success!"
   coverage:
     needs: [ e2e-test-throttling, deny-test ]
+    if: ${{ github.event.pull_request.user.login != 'dependabot[bot]' }}
```

```suggestion
    if: ${{ github.actor != 'dependabot[bot]' }}
```

`github.actor` の方がよさげですがいかがでしょうか？
https://github.com/search?q=github.actor+%21%3D+%27dependabot%5Bbot%5D%27&type=code

---

@nanasess 
ありがとうございます。
こちらだとdependabotのPRに、他の人がpushした場合に起動してしまうかと思ったのですが、如何でしょうか？

---

@dotani1111 なるほど、では `github.event.pull_request.user.login` で良さそうですね